src/linting/checks.jl

```diff
@@ -149,21 +149,20 @@ function func_nargs(x::EXPR)
         if ispunctuation(arg)
             # skip
         elseif is_parameters(arg)
-            for j = 1:length(arg)
-                arg1 = arg[j]
-                if is_kwarg(arg1)
-                    push!(kws, Symbol(CSTParser.str_value(CSTParser.get_arg_name(arg1[1]))))
-                elseif is_binary_call(arg1) && kindof(arg1[2]) === CSTParser.Tokens.DDDOT
```

Comment:
The "super minimal" fix would be to change this line to
```
elseif is_unary_call(arg1) && kindof(arg1[2]) === CSTParser.Tokens.DDDOT
```
right?

Reply:
yeah,  ...